CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Change Log - Power Platform Extension
 
 ## NEXTRELEASE
```

Comment:
are we now ready to update this version?

Reply:
Pretty much.
Still need to roll another version of PAC with the regression fixes, and make sure the PowerPages team updates their release notes if they have anything to add.

Reply:
Updated